src/SFML/Window/Unix/GlxContext.cpp

```diff
@@ -1,7 +1,7 @@
 ////////////////////////////////////////////////////////////
 //
 // SFML - Simple and Fast Multimedia Library
-// Copyright (C) 2007-2017 Laurent Gomila (laurent@sfml-dev.org)
+// Copyright (C) 2007-2018 Laurent Gomila (laurent@sfml-dev.org)
```

Comment:
The `2.4.x` branch uses 2017, let's keep it that way.
Could you revert all the year changes? If possible with `git commit --amend` to keep 1 commit 🙂